### DISCUSSION TITLE
How do I change the order of the tabs?

### DISCUSSION THREAD

**Discussion by msueping** [vip-git/react-jsonschema-form-material-ui#399] [Q&A]
I've been looking at this React component with great interest because it seems to be the only one that I've seen that renders material ui controls with functionality that I'm after "out of the box". e.g., the various ways to show a multiselect as demonstrated on the playground. Nice job. 

I decided to give this a try and started with a Tabs UI control. I discovered that when a tab control is used to render my json schema, it reads the json top to bottom and places the tab titles from left to right. I it possible for me to define/override (in the uischema?) the order of the tabs and their content? For example, in my last screen shot, I want to put the "Profile" tab (currently at the right most position) to be between the "Identification" and "Contact" tab

Also, I'm discovering that when I choose a new tab, nothing happens. I'm expecting the data that is under that part of the schema hierarchy to be rendered. The focus never leaves the first tab and its content. Is there something that I need to be setting up for this to work?

Lastly (for now :)), how can I make the first tab "disappear", i.e., it is currently showing an internally generated ID that is meaningless to the user and as such, I'd rather not have it displayed. Also notice how it doesn't show me an actual ID value but instead show an "Object". Why? I can't see what the other tab content looks like (assuming there is some content getting rendered) because I can't change to a another tab!!!

My schema looks like this
![image](https://user-images.githubusercontent.com/67658122/133859719-c5ba688e-6c9a-49df-9b33-5cd82cba08ee.png)

The UIschema looks like this
![image](https://user-images.githubusercontent.com/67658122/133859779-c03fecd9-69e5-41e0-92b5-f194489d0c6a.png)

And results in
![image](https://user-images.githubusercontent.com/67658122/133859844-da0820b9-9820-4fa1-9746-ae32eb8d9cfd.png)

**Comment by vip-git** [comment]
Hi @msueping - Firstly thanks for trying out `react-jsonschema-form-material-ui` .

Is it possible to show your complete `json-schema` and `ui-schema` to inspect what can be wrong here ?

Currently from screenshot it looks like you are missing the properties tab.

To change the order of tabs - its all relative to the order of your properties itself 1st level.
